.github/workflows/test-package.yml

```diff
@@ -331,7 +331,7 @@ jobs:
       - run: flutter build apk
         working-directory: ./jnigen/example/notification_plugin/example
       - name: re-generate bindings
-        run: flutter pub run jnigen -Ddart_root=_dart -Dc_root=_c --config jnigen.yaml
+        run: flutter pub run jnigen -Doutput.dart.root=_dart -Doutput.c.root=_c --config jnigen.yaml
```

Comment:
Should we call it `path` maybe instead of `root`? A path is necessarily a root if we output more than one file. And a path makes more sense if we output only a single file.

Reply:
Should this be changed in the PR adding single file support or in current PR? Currently root is unambiguous and communicates it's a directory.

Reply:
We're changing the API already in this PR. So if we do it now, we don't have to change the API twice.